include/freerdp/codec/h264.h

```diff
@@ -60,6 +70,11 @@ extern "C"
 	                                  const RECTANGLE_16* regionRect, BYTE** ppDstData,
 	                                  UINT32* pDstSize, RDPGFX_H264_METABLOCK* meta);
 
+	/* API for user to fill YUV I420 buffer before encoding */
+	FREERDP_API INT32 avc420_get_yuv_buffer(H264_CONTEXT* h264, UINT32 nSrcStride, UINT32 nSrcWidth,
```

Comment:
please rename, these functions do normal H264 stuff and not specific AVC420/AVC444 RDP mode stuff

Reply:
Done